website/settings/local-dist.py

```diff
@@ -108,11 +112,6 @@ class CeleryConfig(defaults.CeleryConfig):
 
 USE_CDN_FOR_CLIENT_LIBS = False
 
-# WARNING: `SENDGRID_WHITELIST_MODE` should always be True in local dev env to prevent unintentional spamming.
-# Add specific email addresses to `SENDGRID_EMAIL_WHITELIST` for testing purposes.
-SENDGRID_WHITELIST_MODE = True
```

Comment:
@mfraezz I checked with another member of the cloud team, but just want to confirm with you this isn't still being used somewhere.